elements/pfe-content-set/README.md

```diff
@@ -3,35 +3,27 @@
 
 ## Overview
 
-`pfe-content-set` is a combo component, which brings together the utilities of the accordion and tabs components. Effectively both of these components do the same job, which is to encapsulate chunks of information under headings for easier browsing. Hiding some information and allowing the user to toggle through the headings to show other bits of information. 
+`pfe-content-set` is a combo component, which brings together the utilities of the accordion and tabs components. Effectively both of these components do the same job, which is to encapsulate chunks of information under headings for easier browsing. Hiding some information and allowing the user to toggle through the headings to show other bits of information.
 
 Since tabs can pose a layout issue on mobile because of the lack of horizontal space, this component will first assess the width of the parent container. If the width is less than 768px, the component will render the content within the `<pfe-accordion>` component. If it is larger than this value, the content will be rendered inside the `<pfe-tabs>` component.
 
 ## Dependencies
 
 Requires both the `pfe-accordion` and `pfe-tabs` components, as well as the base `pfelement`.
 
-
 ## Usage
 
-You must wrap each chunk of information (heading + content) in a `<pfe-content-set-group>` tag. 
-
-You must also apply the `pfe-heading` to the HTML heading element, like an H2 or H3. Otherwise content within the component will not be rendered.
+<!-- You must wrap each chunk of information (heading + content) in a `<pfe-content-set-group>` tag. -->
```

Comment:
Can this comment be deleted now?  It appears not to be needed anymore.

Reply:
@castastrophe I have removed that.